.github/workflows/ci.yml

```diff
@@ -7,7 +7,7 @@ jobs:
     strategy:
       fail-fast: false
       matrix:
-        node-version: [0.12.x, 4.x, 14.x, 16.x, 18.x]
+        node-version: [12.x, 14.x, 16.x, 18.x]
```

Comment:
Only >= 12.x compiles on macOS m1

At the least, leave a comment about this in the yml above